doc/en/user/source/community/features-templating/directives.rst

```diff
@@ -0,0 +1,1211 @@
+            <st:value>25.0</st:value>
+          </st:Temperature>
+        </st:temperature>
+      </st:temperature>
+      <st:pressure>
+        <st:pressure>
+          <st:Pressure>
+            <st:time>2016-12-19T11:30:26.000Z</st:time>
+            <st:value>1019.0</st:value>
+          </st:Pressure>
+        </st:pressure>
+        <st:pressure>
+          <st:Pressure>
+            <st:time>2016-12-19T11:30:51.000Z</st:time>
+            <st:value>1015.0</st:value>
+          </st:Pressure>
+        </st:pressure>
+      </st:pressure>
+      <st:wind_speed>
+        <st:wind_speed>
+          <st:Wind_speed>
+            <st:time>2016-12-19T11:29:24.000Z</st:time>
+            <st:value>80.0</st:value>
+          </st:Wind_speed>
+        </st:wind_speed>
+      </st:wind_speed>
+    </st:MeteoStations>
+  </wfs:member>
+ </wfs:FeatureCollection>
+
+
+The following GML template will produce the above output:
+
+.. code-block:: xml
+
+  <gft:Template>
+  <gft:Options>
+    <gft:Namespaces xmlns:st="http://www.stations.org/1.0"/>
+  </gft:Options>
+  <st:MeteoStations gml:id="${@id}">
+  <st:code>$${strConcat('Station_',st:code)}</st:code>
+  <st:name>${st:common_name}</st:name>
+  <st:geometry>${st:position}</st:geometry>
+  <st:temperature gft:isCollection="true" gft:source="st:meteoObservations/st:MeteoObservationsFeature" gft:filter="xpath('st:meteoParameters/st:MeteoParametersFeature/st:param_name') = 'temperature'">
+  <st:Temperature>
+    <st:time>${st:time}</st:time>
+    <st:value>${st:value}</st:value>
+  </st:Temperature>
+  </st:temperature>
+  <st:pressure gft:isCollection="true" gft:source="st:meteoObservations/st:MeteoObservationsFeature"  gft:filter="xpath('st:meteoParameters/st:MeteoParametersFeature/st:param_name') = 'pressure'">
+  <st:Pressure>
+    <st:time>${st:time}</st:time>
+    <st:value>${st:value}</st:value>
+  </st:Pressure>
+  </st:pressure>
+  <st:wind_speed gft:isCollection="true" gft:source="st:meteoObservations/st:MeteoObservationsFeature"  gft:filter="xpath('st:meteoParameters/st:MeteoParametersFeature/st:param_name') = 'wind speed'">
+  <st:Wind_speed>
+    <st:time>${st:time}</st:time>
+    <st:value>${st:value}</st:value>
+  </st:Wind_speed>
+  </st:wind_speed>
+  </st:MeteoStations>
+ </gft:Template>
+
+
+In the GML case :code:`filter` and :code:`source` directives are defined in a slightly different manner from the JSON usecase.
+
+* The filter needs to be defined as an attribute :code:`gft:filter` in the element that is meant to be filtered.
+* The source needs to be defined as an attribute :code:`gft:source` in the element that will set the source for its child elements.
+* The attribute :code:`gft:isCollection="true"` define a directive meant to be used in GML templates to mark collection elements: this directive is needed since xml doesn't have the array concept and the template mechanism needs to be informed if an element should be repeated because it represent a collection element. 
```

Comment:
```suggestion
* The attribute :code:`gft:isCollection="true"` defines a directive meant to be used in GML templates to mark collection elements: this directive is needed since XML doesn't have the array concept and the template mechanism needs to be informed if an element should be repeated because it represent a collection element. 
```